deltachat-ios/Controller/AccountSetupController.swift

```diff
@@ -666,6 +660,8 @@ class AccountSetupController: UITableViewController, ProgressAlertHandler {
                 } else if ui["done"] as! Bool {
                     self.dcContext.maybeStartIo()
                     self.updateProgressAlertSuccess(completion: self.handleLoginSuccess)
+                    self.emailCell.setText(text: nil)
```

Comment:
Why exactly is this needed?

I find it a bit weird to see the credential fields to be cleared right before being redirected.
If the clearing is necessary I think we can clear it in `handleLoginSuccess `, so it will be executed in the completion handler.

Reply:
Thanks, it was indeed not needed.